src/features/PesterTests.ts

```diff
@@ -87,6 +91,14 @@ export class PesterTestsFeature implements IFeature {
         // special chars like & $ @ () [], we do have to double up the interior single quotes.
         const scriptPath = uri.fsPath.replace(/'/g, "''");
 
+        let pesterOption: string;
+        if (pesterSupportsLineNumber) {
```

Comment:
Since you've made `describeBlockLineNumber` nullable, do we need `pesterSupportsLineNumber`?  Seems like a null `describeBlockLineNumber` would indicate that the version of Pester in use doesn't support line number invocation.

Reply:
Yes, we could do that, I chose this approach just to make code more readable but can definitely change it